test/test_nn.py

```diff
@@ -1580,6 +1580,15 @@ def test_Sequential_delitem(self):
         del n[1::2]
         self.assertEqual(n, nn.Sequential(l1, l3))
 
+    def test_Sequential_add(self):
+        l1 = nn.Linear(10, 20)
```

Comment:
Can you use smaller sizes here, 1, 2, 3, 4 is fine.

Reply:
Hi, @albanD! Sure, I've updated the tests. Thanks for your input!